scripts/std_msgs_msg_string_benchee.exs

```diff
@@ -0,0 +1,50 @@
+alias Rclex.Pkgs.StdMsgs
+
+struct = %Rclex.Pkgs.StdMsgs.Msg.String{
+  data: for(_ <- 1..(2 ** 16), into: <<>>, do: <<0x30>>)
+}
+
+Benchee.run(%{
+  "create!/0" => {
+    fn ->
+      StdMsgs.Msg.String.create!()
+    end,
+    after_each: fn message ->
+      StdMsgs.Msg.String.destroy!(message)
+    end
+  },
+  "set!/2" => {
+    fn message ->
+      StdMsgs.Msg.String.set!(message, struct)
+      message
+    end,
+    before_each: fn _ ->
+      StdMsgs.Msg.String.create!()
+    end,
+    after_each: fn message ->
+      StdMsgs.Msg.String.destroy!(message)
+    end
+  },
+  "get!/1" => {
+    fn message ->
+      StdMsgs.Msg.String.get!(message)
+      message
+    end,
+    before_each: fn _ ->
+      message = StdMsgs.Msg.String.create!()
+      StdMsgs.Msg.String.set!(message, struct)
+      message
+    end,
+    after_each: fn message ->
+      StdMsgs.Msg.String.destroy!(message)
+    end
+  },
+  "destroy!/1" => {
+    fn message -> StdMsgs.Msg.String.destroy!(message) end,
+    before_each: fn _ ->
+      message = StdMsgs.Msg.String.create!()
+      StdMsgs.Msg.String.set!(message, struct)
+      message
+    end
+  }
+})
```

Comment:
Actually I'm not sure what the role of this file is, but it may be good to prepare this file in [rclex_bench](https://github.com/rclex/rclex_bench) if this is for the evaluation.

Reply:
I put the result of this script to this PR's meme. `mix run scripts/std_msgs_msg_string_benchee.exs`.
In my opinion, I'd like to use these scripts when I develop. So it's useful these are in rclex's repo.

Reply:
Understood, I think it's acceptable to add this file to this repository!

This is the same as what I heard about [iwyu](https://github.com/rclex/rclex/pull/364#issuecomment-2693125354), but others may be wondering what the role of files in `scripts/`, and whether users need to be aware of them. So my suggestion is that it may be good to add notes about these roles and their usage for developers in `scripts/README` or in comments in these scripts. Of course, you don't need to address my suggestion at this time, but please consider working on this on future PRs.

Reply:
I added scripts/README.md, thanks!